packages/e2e-tests/scripts/loadRecording.ts

```diff
@@ -0,0 +1,41 @@
+import { SimpleProtocolClient } from "@replayio/protocol";
+import WebSocket from "ws";
+
+import { logAnimated } from "./log";
+
+const DISPATCH_URL =
+  process.env.DISPATCH_ADDRESS ||
+  process.env.NEXT_PUBLIC_DISPATCH_URL ||
+  "wss://dispatch.replay.io";
+
+const callbacks: any = {
+  onClose: console.log,
+  onError: console.log,
+};
+
+export async function loadRecording(recordingId: string) {
+  const { completeLog, updateLog } = logAnimated(`Processing recording ${recordingId}`);
+
+  const onProgress = ({
+    progressPercent,
+    recordingId,
+  }: {
+    progressPercent: number;
+    recordingId: string;
+  }) => {
+    updateLog(`Processing recording ${recordingId} ${progressPercent}%`);
+  };
+
+  const client = new SimpleProtocolClient(new WebSocket(DISPATCH_URL), callbacks, console.log);
+  client.addEventListener(
+    // @ts-expect-error - when we update protocol client to 0.68, it will throw an error - just remove this comment
+    "Recording.processRecordingProgress",
+    onProgress
+  );
+
+  await client.sendCommand("Recording.processRecording", {
+    recordingId,
+  });
```

Comment:
I think we need to create isolated clients for each recording, else if we _were_ to try batching this, separate recordings would interfere with each other and the function would resolve prematurely.